README.md

```diff
@@ -22,4 +22,54 @@ Testing is done via [pytest](pytest.org):
 
     pytest
 
-All tests should pass before landing on `master`.
\ No newline at end of file
+All tests should pass before landing on `master`.
+
+# Client<->Server Schema
+
+```
+Place: {
+    version: #,
+    id: "<yelp-id>",
+
+    name: "",
+    description: {
+        provider: "wikipedia",
+        text: ""
+    },
+    url: "",
+    phone: "", // consistent format
+
+    address: "",
+    coordinates: {
+        lat: #,
+        lng: #
+    },
+    categories: [{
+        id: "",
+        text: ""
+    }],
+
+    providers: {
+        yelp: {
+            rating: #, // normalized 0-5 w/ half values
+            totalReviewCount: #,
+            url: ""
+        } // ...
+    },
+
+    images: [{
+        src: "", // url to image
+        provider: "", // name to attribute on client
+        providerURL: "" // e.g. click target for image
+    }],
+
+    hours: {
+        monday: [
+            ["6:30", "18:00"]
+        ],
```

Comment:
This hours format should be ok, but there is work to be done to translate this from the Yelp format to the desired format.

The Factual API does provide opening hours in this format, but is less reliable.